ReactWindows/ReactNative/Views/Text/ReactTextViewManager.cs

```diff
@@ -24,6 +26,27 @@ public override string Name
         }
 
         /// <summary>
+        /// Sets the text alignment.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="textAlign">The text alignment string.</param>
+        [ReactProp(ViewProps.TextAlign)]
```

Comment:
What is the affect on the virtual text nodes (not sure if there's an "IsVirtual" check before applying ReactProps (and there are no other ReactProps in this view manager to refute that).